proof/invariant-abstract/ARM_HYP/ArchRetype_AI.thy

```diff
@@ -769,7 +766,7 @@ sublocale retype_region_proofs_gen?: retype_region_proofs_gen
 end
 
 
-context Arch begin global_naming ARM_HYP (*FIXME: arch-split*)
+context Arch begin arch_global_naming (*FIXME: arch-split*)
```

Comment:
Less clear here, but I don't think there is anything that should be fixed here, so probably also to be dropped.